src/ControlSystem/Controller.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+
+/// \ingroup ControlSystemGroup
+/// A PND (proportional to Q and N derivatives of Q) controller that computes
+/// the control signal:
+/// \f[ U(t) = \sum_{k=0}^{N} a_{k} \frac{d^kQ}{dt^k} \f],
+/// where N is specified by the template parameter `deriv_order`.
+
+/// If an averager is used for `q_and_derivs` (as we typically do), there is an
+/// induced time offset, \f$\Delta t\f$, due to the time-weighted averaging.
+/// Therefore, the q_and_derivs that we have in hand are at some time
+/// \f$t_{o}\f$. However, we desire q_and_derivs at the current time
```

Comment:
Can you chance `t_o` to `t_0`? Unless there's some reason that it's `o` as opposed to `0`.